server/rpc.go

```diff
@@ -252,26 +258,29 @@ func genOrchestratorReq(b common.Broadcaster) (*net.OrchestratorRequest, error)
 func getOrchestrator(orch Orchestrator, req *net.OrchestratorRequest) (*net.OrchestratorInfo, error) {
 	addr := ethcommon.BytesToAddress(req.Address)
 	if err := verifyOrchestratorReq(orch, addr, req.Sig); err != nil {
-		return nil, fmt.Errorf("Invalid orchestrator request (%v)", err)
+		return nil, fmt.Errorf("Invalid orchestrator request err=%v", err)
 	}
 
-	webhookRes, err := authenticateBroadcaster(addr.Hex())
-	if err != nil {
-		return nil, fmt.Errorf("authentication failed: %v", err)
+	if _, err := authenticateBroadcaster(addr.Hex()); err != nil {
+		return nil, fmt.Errorf("authentication failed err=%v", err)
```

Comment:
I don't think we need to use the err= convention unless we're logging something. Otherwise if we also use the this convention when these errors are logged we'll end up with log statements that look like `err=...err=` which is a bit noisy.